src/iop/cacorrect.c

```diff
@@ -281,7 +281,15 @@ void process(
   float *RawDataTmp = NULL;
   float *Gtmp = NULL;
 
-  float *out = dt_alloc_align_float(roi_in->width * roi_in->height);
+  const size_t width = roi_in->width;
+  const size_t height = roi_in->height;
+  const size_t ibsize = dt_round_size(width, DT_CACHELINE_BYTES) * (height + 2);
```

Comment:
DT_CACHELINE_FLOATS would actually be sufficient here, but it's not that big an increase in memory allocation to use \_BYTES

Reply:
Right :-) fixed